CMakeLists.txt

```diff
@@ -31,52 +30,50 @@ if(${CMAKE_GENERATOR} STREQUAL "Ninja")
     endif()
 endif()
 
-# Default Blizzard Community Platform API key
-if (DEFINED SC_DEFAULT_APIKEY)
-    add_compile_options(-DSC_DEFAULT_APIKEY="${SC_DEFAULT_APIKEY}")
+# Install paths
+if (SC_USE_FLAT_INSTALL)
+  set(SIMC_INSTALL_BIN ".")
+  set(SIMC_INSTALL_SHARED ".")
+else()
+  set(SIMC_INSTALL_BIN "bin")
+  set(SIMC_INSTALL_SHARED "share/SimulationCraft/SimulationCraft")
 endif()
 
-# output a compile_commands.json used for editor tooling
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Is this the default now? There is no reason not to do this. And considering the quality of cmake docs I don't really want to check, so I'd leave this.

Reply:
Well... kind of. Theoretically sure, a user can pass in `-DCMAKE_EXPORT_COMPILE_COMMANDS=ON` when invoking CMake if they need it.
Realistically though, considering the amount of tooling that can make use of it (or even *requires* it) currently (e.g. everything clang based requires it, vscode cpp plugin works miles better with it) and no downsides to having it on by default you may as well always generate it and make life easier for everyone.